ga4gh/datamodel/reads.py

```diff
@@ -11,37 +11,34 @@
 
 import pysam
 
-import ga4gh.protocol as protocol
 import ga4gh.datamodel as datamodel
+import ga4gh.datamodel.datasets as datasets
 import ga4gh.exceptions as exceptions
+import ga4gh.protocol as protocol
 
 
-class CompoundReadGroupSetId(datamodel.CompoundId):
+class ReadGroupSetCompoundId(datasets.DatasetCompoundId):
```

Comment:
name change to have the 'compound' part after the class type is probably a good idea